src/NavigationService.js

```diff
@@ -0,0 +1,21 @@
+import { NavigationActions } from 'react-navigation';
+
+let _navigator;
+
+function setTopLevelNavigator(navigatorRef) {
+  _navigator = navigatorRef;
+}
+
+function navigate(routeName, params) {
```

Comment:
Yes.

We are calling `setTopLevelNavigator` on app startup (App.js, L#347), so it won't be a problem for now

It might be a problem for instance when we implement push notifications that do something when they are clicked, like navigating to a specific screen (tx detail maybe). We could then implement some kind of queue that will be called in order after the `setTopLevelNavigator` method is called. But I think the current implementation is OK for now. What do you think?

Reply:
Yeah, I agree it's good